lib/rules/lines-around-comment.js

```diff
@@ -115,6 +125,11 @@ module.exports = function(context) {
             after = false;
         }
 
+        // if block start is allowed and comment is at block start then return
+        if (options.allowBlockStart && isCommentAtBlockStart()) {
```

Comment:
Thats true but I think there would be arguments both ways here. Saying it should allow that as it start of the block.
I am not sure what the right answer is here. Thoughts?